elements/noflo-component-editor.html

```diff
@@ -304,6 +333,9 @@
         if (this.component.language === 'javascript') {
           return this.$.JavaScriptExample.textContent.trim();
         }
+        if (this.component.language === 'es6') {
```

Comment:
What about jsx? Also, ES6 and ES7 are called ES2015 now AFAIK.
Atom uses babel if it sees "use babel" in the code. Cf. http://blog.atom.io/2015/02/04/built-in-6to5.html

Cf. my comment here: https://github.com/noflo/noflo/pull/294#discussion-diff-38762552